pages/tools/fileconverter.vue

```diff
@@ -41,7 +41,7 @@
         <cloud-loading />
       </v-col>
 
-      <v-col v-else>
+      <v-col v-else class="pb-5">
```

Comment:
is "pb-5" still needed with the new v-col usage?

Reply:
It's a padding bottom for the stepper, the goal is that the versions aren't right bellow it

Reply:
Or, I could add a padding top to the versions v-col...